<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_chooser</class>
 <widget class="QDialog" name="Dialog_chooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Option</string>
  </property>
  <widget class="QLabel" name="label1_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>195</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>22</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame1_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>25</y>
     <width>195</width>
     <height>90</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="comboBox_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>175</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ok">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
